src/main/java/duke/commands/IncorrectCommand.java

```diff
@@ -0,0 +1,39 @@
+package duke.commands;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents an incorrect command.
+ */
+public class IncorrectCommand extends Command {
```

Comment:
Instead of using a class for an incorrect command, perhaps you could throw an exception instead